src/azure-cli/azure/cli/command_modules/dls/_client_factory.py

```diff
@@ -26,12 +26,8 @@ def cf_dls_filesystem(cli_ctx, account_name):
     from azure.datalake.store import core
     from azure.cli.core._profile import Profile
 
-    profile = Profile(cli_ctx=cli_ctx)
-    subscription_id = None
```

Comment:
It is weird `subscription_id` is set to `None` and then passed to `get_login_credentials()`.

Reply:
So I removed this